src/main/scala/com/fulcrumgenomics/umi/CallDuplexConsensusReads.scala

```diff
@@ -86,8 +86,9 @@ class CallDuplexConsensusReads
  @arg(flag='1', doc="The Phred-scaled error rate for an error prior to the UMIs being integrated.") val errorRatePreUmi: PhredScore = DefaultErrorRatePreUmi,
  @arg(flag='2', doc="The Phred-scaled error rate for an error post the UMIs have been integrated.") val errorRatePostUmi: PhredScore = DefaultErrorRatePostUmi,
  @arg(flag='m', doc="Ignore bases in raw reads that have Q below this value.") val minInputBaseQuality: PhredScore = DefaultMinInputBaseQuality,
- @arg(flag='t', doc="If true quality trim input reads in addition to masking low Q bases.") val trim: Boolean = false,
- @arg(flag='S', doc="The sort order of the output, if `:none:` then the same as the input.") val sortOrder: Option[SamOrder] = Some(SamOrder.Queryname)
+ @arg(flag='t', doc="If true, quality trim input reads in addition to masking low Q bases.") val trim: Boolean = false,
+ @arg(flag='S', doc="The sort order of the output, if `:none:` then the same as the input.") val sortOrder: Option[SamOrder] = Some(SamOrder.Queryname),
+ @arg(flag='s', doc="If true, output consensus reads generated from only one strand, otherwise require both strands.") val allowSingleStrand: Boolean = false
```

Comment:
I wonder if it would be more consistent here to turn this into:

```scala
@arg(flag='M', minElements=1, maxElements=3, doc="The minimum number of input reads to a consensus read.") val minReads: Seq[Int] = Seq(1)
```

So that it more directly matches what is available in `FilterConsensusReads` and provides a bit more control?  E.g. you might be willing to take any consensus from at least 2 reads, but not single-strands from 1 read?